client/my-sites/themes/theme-options.js

```diff
@@ -45,7 +46,7 @@ const buttonOptions = {
 		// We don't know where support docs for a given theme on a self-hosted WP install are,
 		// and free themes don't have support docs.
 		isHidden: ( site, theme ) => ( site && site.jetpack ) || ! Helper.isPremium( theme )
-	},
+	}
```

Comment:
eslint does not produce any errors with the trailing comma, why remove it? I thought that the trailing comma is removed by babel, and I can't find anything about it in our coding standards, did I miss some discussion about it?